grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -66,6 +67,15 @@ class InventoryItemController {
             // now populate the rest of the commmand object
             inventoryService.getStockCardCommand(cmd, params)
 
+            List<ProductPackage> packages = []
```

Comment:
Let's add this to the getStockCardCommand() method.

Reply:
Actually we should add this logic as a convenience method on Product. 
```
List getUoms() { 
  // do a thing, return a List of Maps or DTOs
  return productPackages.collect { [uom: it.uom, quantity: it.quantity] }.unique()

  // or use one of the unique methods below to do it in the other direction 
  //return productPackages.unique { ... }.collect { [uom: it.uom, quantity: it.quantity ] }
  //return productPackages.unique(new UomComparator()).collect { [uom: it.uom, quantity: it.quantity ] }
}
```

https://docs.groovy-lang.org/latest/html/groovy-jdk/java/util/Collection.html#unique(groovy.lang.Closure)
https://docs.groovy-lang.org/latest/html/groovy-jdk/java/util/Collection.html#unique(java.util.Comparator)